src/sidebar/store/modules/groups.js

```diff
@@ -156,44 +180,22 @@ const getMyGroups = createSelector(
   }
 );
 
-/**
- * Return groups the user isn't a member of that are scoped to the URI.
- *
- * @return {Group[]}
- */
-const getFeaturedGroups = createSelector(
-  state => state.groups.groups,
-  groups => groups.filter(group => !group.isMember && group.isScopedToUri)
-);
-
 /**
  * Return groups that don't show up in Featured and My Groups.
  *
  * @return {Group[]}
  */
 const getCurrentlyViewingGroups = createSelector(
-  allGroups,
-  getMyGroups,
-  getFeaturedGroups,
+  state => allGroups(state.groups),
```

Comment:
Can you walk me through this change? BTW, alternate API to `createSelector` lets you pass input selectors as an array in a single argument: https://github.com/reduxjs/reselect#api — I find this a little easier to read but it is of course developer preference here...

Did this work before with just the reference to the selectors? Curious.

Reply:
I honestly didn't think about this too much but just adapted the existing code to work with the fact that this selector receives root state as input but calls other selectors that expect the module state as input.